### PR TITLE
Add klantendag event and RSVP page

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Fix form alignment
- [x] Test form capture and automatic mail (in FormKeep)

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Michel is in Germany this week, but this klantendag campaign needs to be launched. 
When this is ready I'll let it know.

File: locales/nl.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (fatboypunk) [inline locales/nl.yml]: This isn't needed anymore?

File: locales/nl.yml [hunk 2]
```diff
@@ -217,16 +217,16 @@ nl:
...
-    phone_link: '+31503144832'
+    phone_link: "+31503144832"
   elearning:
     heading: E-Learning ontwikkelen met Defacto
   elearning_ebook:
     heading_html: Ontvang gratis ons <span class="pre">E-Learning</span> Handboek
     image_alt: E-Learning Handboek Leren & Laten Leren
   error:
-    heading: 'Sorry, deze pagina bestaat niet.'
+    heading: "Sorry, deze pagina bestaat niet."
```
Comment (fatboypunk) [inline locales/nl.yml]: We're changing  `'` to `"` but do we also care bout no surrounding quotes vs surrounding quotes?

Comment (RonaldDijkstra) [reply locales/nl.yml]: Oh shit, some autoformatting going on here.

**Comment by RonaldDijkstra** [comment]
Ready

File: locales/nl.yml [hunk 3]
```diff
@@ -248,14 +248,14 @@ nl:
...
-    choose_event: Kies een plaats & datum
+    add_coworker: Meld extra collega aan
+    add_coworker_button: + Meld extra collega aan
+    choose_event: Kies een sessie
     email: E-mail
     first_name: Voornaam
     message: Wat kunnen we voor je betekenen?
     message_for_organisation: Opmerkingen voor de organisatie
-    message_questions_diet: Vragen die je wilt bespreken (vermeld hier ook eventuele dieetwensen)
+    message_plus_diet: Opmerkingen/dieetwensen
```
Comment (fatboypunk) [inline locales/nl.yml]:
```suggestion
    message_plus_diet: Opmerkingen en of dieetwensen
```

Comment (RonaldDijkstra) [reply locales/nl.yml]: Nice

**Pull request merged**